test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -494,15 +494,14 @@ export default class PerfCompare extends Component {
                     let runBaseline = this.state.benchmarkRuns.benchmarkRunBaseline;
                     let runTest = this.state.benchmarkRuns.benchmarkRunTest
                     curRawValues = {
-                        "baseline": runBaseline.parsedVariants[i].metrics[j].value,
-                        "test": runTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex].value
+                        "baseline": Object.entries(runBaseline.parsedVariants[i].metrics[j]),
+                        "test": Object.entries(runTest.parsedVariants[curMatchingTestVariantIndex].metrics[curMatchingTestMetricIndex])
                     };
                 }
-
-                curBaselineScore = parsedVariantsBaseline.metrics[j].value.mean;
-                curBaselineCI = parsedVariantsBaseline.metrics[j].value.CI;
-                curTestScore = parsedVariantsTest.metrics[j].value.mean;
-                curTestCI = parsedVariantsTest.metrics[j].value.CI;
+                curBaselineScore = parsedVariantsBaseline.metrics[j]['mean'];
```

Comment:
This would affect non-Perffarm runs as well. Make sure to test both Perffarm and non-Perffarm runs to avoid breaking things.